src/locale/locales/nl/messages.po

```diff
@@ -0,0 +1,7903 @@
+
+#: src/screens/Signup/StepInfo/Policies.tsx:48
+msgid "By creating an account you agree to the <0>Terms of Service</0> and <1>Privacy Policy</1>."
+msgstr "Door een account aan te maken stemt u in met de <0>Servicevoorwaarden</0> en het <1>Privacybeleid</1>."
+
+#: src/components/LabelingServiceCard/index.tsx:62
+msgid "By {0}"
+msgstr "Door {0}"
+
+#: src/view/com/composer/photos/OpenCameraBtn.tsx:72
+msgid "Camera"
+msgstr "Camera"
+
+#: src/view/com/modals/AddAppPasswords.tsx:180
+msgid "Can only contain letters, numbers, spaces, dashes, and underscores. Must be at least 4 characters long, but no more than 32 characters long."
+msgstr "Kan alleen letters, cijfers, spaties, streepjes en onderstrepingstekens bevatten. Moet minimaal 4 tekens lang zijn, maar niet meer dan 32 tekens."
+
+#: src/components/Menu/index.tsx:235
+#: src/components/Prompt.tsx:129
+#: src/components/Prompt.tsx:131
+#: src/components/TagMenu/index.tsx:267
+#: src/screens/Deactivated.tsx:161
+#: src/screens/Profile/Header/EditProfileDialog.tsx:220
+#: src/screens/Profile/Header/EditProfileDialog.tsx:228
+#: src/view/com/composer/Composer.tsx:684
+#: src/view/com/modals/ChangeEmail.tsx:213
+#: src/view/com/modals/ChangeEmail.tsx:215
+#: src/view/com/modals/ChangeHandle.tsx:141
+#: src/view/com/modals/ChangePassword.tsx:268
+#: src/view/com/modals/ChangePassword.tsx:271
+#: src/view/com/modals/CreateOrEditList.tsx:335
+#: src/view/com/modals/CropImage.web.tsx:97
+#: src/view/com/modals/InAppBrowserConsent.tsx:75
+#: src/view/com/modals/InAppBrowserConsent.tsx:77
+#: src/view/com/modals/LinkWarning.tsx:105
+#: src/view/com/modals/LinkWarning.tsx:107
+#: src/view/com/modals/VerifyEmail.tsx:255
+#: src/view/com/modals/VerifyEmail.tsx:261
+#: src/view/com/util/post-ctrls/RepostButton.tsx:166
+#: src/view/screens/Search/Search.tsx:910
+msgid "Cancel"
+msgstr "Annuleer"
+
+#: src/view/com/modals/CreateOrEditList.tsx:340
+#: src/view/com/modals/DeleteAccount.tsx:174
+#: src/view/com/modals/DeleteAccount.tsx:296
+msgctxt "action"
+msgid "Cancel"
+msgstr "Annuleer"
+
+#: src/view/com/modals/DeleteAccount.tsx:170
+#: src/view/com/modals/DeleteAccount.tsx:292
+msgid "Cancel account deletion"
+msgstr "Annuleeer accountverwijdering"
+
+#: src/view/com/modals/ChangeHandle.tsx:137
+msgid "Cancel change handle"
+msgstr "Annuleer wijzigen handle"
+
+#: src/view/com/modals/CropImage.web.tsx:94
+msgid "Cancel image crop"
+msgstr "Annuleer afbeelding bijsnijden"
+
+#: src/view/com/util/post-ctrls/RepostButton.tsx:161
+msgid "Cancel quote post"
+msgstr "Annuleer citaatbericht"
+
+#: src/screens/Deactivated.tsx:155
+msgid "Cancel reactivation and log out"
+msgstr "Annuleer heractivering en meld af"
+
+#: src/view/com/modals/ListAddRemoveUsers.tsx:88
+#: src/view/screens/Search/Search.tsx:902
+msgid "Cancel search"
+msgstr "Annuleer zoeken"
+
+#: src/view/com/modals/LinkWarning.tsx:106
+msgid "Cancels opening the linked website"
+msgstr "Annuleert openen van de gelinkte website"
```

Comment:
```suggestion
msgstr "Annuleert het openen van de gelinkte website"
```

Reply:
Dito, left unchanged.

Reply:
Ditto response

Reply:
Sorry for jumping in here, but that string is an `accessibilityHint` which I was under the impression should contain "a brief description of the result of performing an action on the accessibility element." (from Apple Developer Documentation) So isn't the English text correct? 🤔

Reply:
Opening of the link has not yet started. That's a technical process that in some cases can actually be cancelled. 

This is more or less a dialog to "ensures you know you're going to leave the current site / app" and that the link you clicked on is shady. `Cancel` is a really bad negative action (because it's often unclear what exactly you're cancelling).

The relevant code is as follows: 

```jsx          
         <Button
            testID="confirmBtn"
            type="primary"
            onPress={onPressVisit}
            accessibilityLabel={share ? _(msg`Share Link`) : _(msg`Visit Site`)}
            accessibilityHint={
              share
                ? _(msg`Shares the linked website`)
                : _(msg`Opens the linked website`)
            }
            label={share ? _(msg`Share Link`) : _(msg`Visit Site`)}
            labelContainerStyle={{justifyContent: 'center', padding: 4}}
            labelStyle={[s.f18]}
          />

          <Button
            testID="cancelBtn"
            type="default"
            onPress={() => {
              closeModal()
            }}
            accessibilityLabel={_(msg`Cancel`)}
            accessibilityHint={_(msg`Cancels opening the linked website`)}
            label={_(msg`Cancel`)}
            labelContainerStyle={{justifyContent: 'center', padding: 4}}
            labelStyle={[s.f18]}
          />
```

Even if we ignore the "pedanticness" of the process of opening, the dialog is doubled for use when you are sharing a link that is shady. Cancel does not cancel opening the linked website. It cancels (stops you from) sharing the link. 

Better actions would have been:

- "Do not share link"
- "Do not open website"

And then neither needs an `accessibilityHint`.